docs/public-networks/how-to/connect/sync-node.md

```diff
@@ -85,8 +85,8 @@ You can observe the `besu_synchronizer_fast_sync_*` and `besu_synchronizer_world
 
 !!! important
 
-    Snap sync is an experimental feature.
-    However, we recommend using snap sync over fast sync even in certain production environments (for example, staking),
+    Snap sync is an early access feature.
```

Comment:
Thanks! Everybody is using Snap and checkpoint, so I would remove the line completely
Approved! :)